app.js

```diff
@@ -0,0 +1,16 @@
+const kaholoPluginLibrary = require("kaholo-plugin-library");
+const { execute } = require("./ansible-cli");
+const { prepareRunPlaybookPayload } = require("./helpers");
+
+function runPlaybook(params) {
+  const payload = prepareRunPlaybookPayload(params);
```

Comment:
`payload` contains pretty much the same data as `params` witch such difference that some of them are parsed or trimmed. Does that change the nature of the data and makes them no longer params?